src/html/html-script-finder.ts

```diff
@@ -44,13 +45,34 @@ export class HtmlScriptFinder implements HtmlEntityFinder {
           entities.push(
               new ImportDescriptor<ASTNode>('html-script', importUrl, node));
         } else {
+          let locationOffset: LocationOffset|undefined = undefined;
```

Comment:
Extracted as `getLocationOffsetOfStartOfTextContent`, and removed unnecessary assignment.